vars.mk

```diff
@@ -1,9 +1,9 @@
 # Pinned Versions
-SNYK_DESKTOP_VERSION=1.790.0
+SNYK_DESKTOP_VERSION=v1.802.0
 SNYK_USER_VERSION=1.460.0
 SNYK_OLD_VERSION=1.382.1
-# Digest of the 1.790.0 snyk/snyk:docker image
-SNYK_IMAGE_DIGEST=sha256:579d7263467ec0cffb44ec1720a320dd1ed6a78d6d4a8d9f58088a26f1d35ac4
+# Digest of the 1.802.0 snyk/snyk:docker image
+SNYK_IMAGE_DIGEST=sha256:bf3b57416af416edec358a184a01f2437f70502bebf0c7a8e787cf68bd6ba428
```

Comment:
small nit. This digest seems to match 1.801.0, the CI pipeline to push a new image takes a bit of time after a new GH release is already visible. 😬

Reply:
Yep that's why it's a draft PR, I'm waiting for the image release on Hub 😉